packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -42,8 +42,16 @@ contract FederatedAttestations is
   mapping(bytes32 => mapping(address => IdentifierOwnershipAttestation[])) public identifierToAddresses;
   // account -> issuer -> identifiers
   mapping(address => mapping(address => bytes32[])) public addressToIdentifiers;
-  // signer => revocation time
-  mapping(address => uint256) public revokedSigners;
+  // signer => isRevoked
+  mapping(address => bool) public revokedSigners;
+
+  bytes32 public constant EIP712_VALIDATE_ATTESTATION_TYPEHASH = keccak256(
+    "IdentifierOwnershipAttestation(bytes32 identifier,address issuer,address account,uint256 issuedOn)"
+  );
+  bytes32 public eip712DomainSeparator;
+
+  // TODO: should this be hardcoded here?
+  bytes32 constant signerRole = keccak256(abi.encodePacked("celo.org/core/attestation"));
```

Comment:
It doesn't seem like it's possible to import it directly from the `Accounts` contract, unless we declare the constant in a library and import it from there instead

Reply:
makes sense! oops sorry I meant like `SIGNER_ROLE` (same style as the typehash)
maybe leave the TODO in so we can double check with more experienced contracts peeps about the best way to handle it since we can't import, up to you!